src/utilities/constants.ts

```diff
@@ -0,0 +1,2 @@
+export const MAXIMUM_FUTURE_DATE = null;
```

Comment:
```suggestion
export const MAXIMUM_FUTURE_DATE = today + 30;
```

// Set maximum future date to a actual dummy future date, Otherwise check might fail. Until we get the correct Future date from EPA.